api/src/routers/past-meals.js

```diff
@@ -0,0 +1,11 @@
+import express from "express";
+import knex from "../database_client.js";
+
+
+const pastMeal = express.Router();
+const currectDay=new Date();
```

Comment:
The `new Date()` returns the current date with time, so it's better to rename it to `now` for example

```suggestion
const now = new Date();
```